### PR TITLE
feat: add ripple effect on the web

### PR DESCRIPTION
**Pull request by satya164**
kapture 2019-01-31 at 21 00 58

### CODE REVIEW

File: src/components/TouchableRipple/index.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      </TouchableWithoutFeedback>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  touchable: ({
+    position: 'relative',
+    overflow: 'hidden',
+  }: any),
```
Comment (jaulz) [inline src/components/TouchableRipple/index.js]: Do we need the type here?

**Comment by jaulz** [comment]
Looks very good! 😊

**Comment by RichardLindhout** [comment]
Nice work guys 👍  
Possible improvement could be to implement focused state but I thinks thats something not implemented in react-native-web yet.

**Comment by satya164** [comment]
@RichardLindhout yeah, unfortunately :(

**Comment by satya164** [comment]
@all-contributors add @satya164 for ideas, code and docs

**Comment by satya164** [comment]
@all-contributors add @satya164 for ideas, code and docs

**Pull request merged**